internal/repos/scheduler.go

```diff
@@ -401,6 +404,12 @@ func (s *updateScheduler) DebugDump() interface{} {
 		data.UpdateQueue = append(data.UpdateQueue, update)
 	}
 
+	var err error
+	data.SyncJobs, err = database.ExternalServices(db).GetSyncJobs(ctx)
+	if err != nil {
+		log15.Warn("Getting external service sync jobs foe debug page", "error", err)
```

Comment:
nit:
```suggestion
		log15.Error("Getting external service sync jobs foe debug page", "error", err)
```

I would argue this is in fact an error, but we decided to not block response to the client.

Reply:
My thinking is that since we still make a best effort to return something to the user it's only a warning.

Reply:
Matter of taste I guess, just nit anyway 😁